pkg/kubelet/cm/container_manager_linux.go

```diff
@@ -841,7 +840,7 @@ func EnsureDockerInContainer(dockerAPIVersion *utilversion.Version, oomScoreAdj
 	type process struct{ name, file string }
 	dockerProcs := []process{{dockerProcessName, dockerPidFile}}
 	if dockerAPIVersion.AtLeast(containerdAPIVersion) {
-		dockerProcs = append(dockerProcs, process{containerdProcessName, containerdPidFile})
```

Comment:
it is a good practice to explain the magic constants, even `""`. Maybe say that in many cases `containerd` is not started by docker and specified using [`--containerd` option](https://docs.docker.com/engine/reference/commandline/dockerd/) so there is no pid file available. See comment here: https://github.com/moby/moby/issues/23069#issuecomment-222270659